modules/010-prometheus-crd/crds/doc-ru-customalertmanager.yaml

```diff
@@ -37,1236 +45,366 @@ spec:
                         key:
                           description: Ключ клиента.
                         insecureSkipVerify:
-                          description: Отключить проверку сертификата.
+                          description: Отключение проверки сертификата.
                 internal:
                   type: object
+                  description: Параметры настройки внутренного Alertmanager.
                   properties:
                     inhibitRules:
-                      description: List of inhibition rules. The rules will only apply to
-                        alerts matching the resource’s namespace.
+                      description: Список правил запрета.
                       items:
-                        description: InhibitRule defines an inhibition rule that allows
-                          to mute alerts when other alerts are already firing. See https://prometheus.io/docs/alerting/latest/configuration/#inhibit_rule
+                        description: Правила запрета используются для подавления уведомлений от определенных оповещений, если уже сработали другие определенные оповещения ([подробнее...](https://prometheus.io/docs/alerting/latest/configuration/#inhibit_rule)).
```

Comment:
I think we do not translate inhibitRule word.

Reply:
Fixed